.github/workflows/continuous-integration.yml

```
@@ -42,11 +39,11 @@ jobs:
       if: steps.composer-cache.outputs.cache-hit != 'true'
       run: composer install --prefer-dist --no-progress --no-suggest --no-interaction
 
-    - name: Run PHP CS fixer
-      run: vendor/bin/php-cs-fixer fix --config=.php-cs-fixer.dist.php --dry-run --diff --verbose --stop-on-violation
```

Normalize is the fastest command, so moved above to fail fast.